sandbox/apparmor/notify/listener/listener.go

```diff
@@ -82,8 +75,8 @@ type Request struct {
 	// Permission is the opaque permission that is being requested.
 	Permission any
 
-	// replyChan is a channel for writing the response.
-	replyChan chan *Response
+	// replyChan is a channel for sending the explicitly allowed permissions.
+	replyChan chan any
```

Comment:
it makes me slightly uneasy to erase all type and pass around `any` here, as if we're plugging a leaky interface and then make it harder to follow the code. Since we already expose types from `notify` package, we may as well introduce a generic notify.Permissions interface with a single method such that `any` doesn't match anymore:

```
// in apparmor/notiry
type Permissions interface {
    Kind() string // eg. returns "file"
    String() string // eg. returns "file:rwxm" 
}
```

In this case we'd have:

```
func AbstractPermissionsToAppArmorPermissions(iface string, permissions []string) (notify.Permissions, error) {
    ...
}
```

and then the listener package can do type assertion to a concrete type. FWIW, I think this can be a followup